src/demo/ui.jsx

```diff
@@ -836,6 +855,7 @@ let Train = class Train extends React.Component {
             }}
             sound={'no'}
           >
+            <FontAwesomeIcon icon={faBan} style={{marginRight: 10}} />
```

Comment:
Worth hoisting these three to a common style?

Reply:
yes!